front/src/app/todo-list/todo-list.component.scss

```diff
@@ -113,12 +110,32 @@
         }
       }
     }
-    .mat-mdc-text-field-wrapper  {
-      height: 26px;
+
+    .mat-mdc-text-field-wrapper {
+      margin-top: 2px;
+      height: 26px !important;
       background-color: rgb(231, 231, 231);
     }
-    .mat-mdc-form-field-subscript-wrapper, .mat-mdc-form-field-focus-overlay, .mdc-line-ripple {
+    .mat-mdc-form-field-flex {
+      margin-top: -10px !important;
+    }
+    .mat-mdc-form-field-focus-overlay, .mdc-line-ripple {
       display: none;
     }
   }
 }
+
+.new-task-list {
+  ::ng-deep {
+    .mat-mdc-text-field-wrapper {
+      height: 50px !important;
+    }
+    .mat-mdc-form-field-flex {
+      margin-top: -3px !important;
+    }
+  }
+}
+

```

Comment:
please change "active group" styles as well 
![image](https://github.com/shanovskyi-b/todo-app/assets/18092448/f806790d-3907-4260-ad6f-56bd06deb4e6)

- make the background color blue (same as the bottom-border color in mat-input)
- make border radius smaller - same as in mat-input or even samller (so that it' no so rounded)